cni/ipam/plugin/main.go

```diff
@@ -23,17 +22,6 @@ func main() {
 	var config common.PluginConfig
 	config.Version = version
 
-	logDirectory := "" // Sets the current location as log directory
```

Comment:
did we validate ipam logs moved to zap

Reply:
yes I did with this PR: https://github.com/Azure/azure-container-networking/pull/2299
Verified azure-ipam.log